Cargo.toml

```diff
@@ -1,17 +1,47 @@
 [package]
 authors = [
+  "Jonas Zaddach <jonas.zaddach@gmail.com",
+  "Zachary Sims <zac@icebergdefender.com>",
   "Scott Hutton <schutton@cisco.com>",
-  "Micah Snyder <micasnyd@cisco.com>",
-  "Jonas Zaddach <jzaddach@cisco.com>",
 ]
 categories = ["api-bindings"]
-description = "Asynchronous API for clamav-sys"
-documentation = "https://docs.rs/clamav-async-rs"
+description = "Async ClamAV bindings for Rust"
 edition = "2021"
-keywords = ["clamav", "libclamav", "antivirus"]
+exclude = ["test_data/*"]
+homepage = "https://github.com/zaddach/clamav-rs"
 license = "GPL-2.0"
 name = "clamav-async"
-repository = "https://github.com/Cisco-Talos/clamav-async-rs"
-version = "0.1.0"
+repository = "https://github.com/zaddach/clamav-rs"
+version = "0.5.5"
+
+[features]
+default = ["native-impl"]
+# native-impl enables native implementations where possible (ignoring
+# implementations provided by libclamav)
+native-impl = []
+# tokio-runtime activates async functionality
 
 [dependencies]
+bitflags = "2"
+derivative = "2"
+lazy_static = "1"
+libc = "0.2"
+log = "0.4"
+thiserror = "1"
+time = { version = "0.3", features = [
+  "parsing",
+  "macros",
+  "formatting",
+  "std",
+] }
+tokio = { version = "1", features = ["sync", "rt", "macros"] }
+tokio-stream = { version = "0.1" }
+
+[dependencies.clamav-sys]
+git = "https://github.com/Cisco-Talos/clamav-sys.git"
+
+[target.'cfg(windows)'.dependencies]
+bindings = { version = "0.5.5", package = "clamav-rs-bindings" }
```

Comment:
It's how Jonas had it.

Yes, it works -- it brings in the `clamav-rs-bindings`, but addressable in the namespace as `bindings::` rather than `clamav_rs_bindings::`

Reply:
Weird.